### PR TITLE
add sunDAI

### PR DESCRIPTION
**Pull request by johannbarbie**
added sundai contract on rinkeby and changed colors

### CODE REVIEW

**Comment by troggy** [comment]
added token config by domain.

sundai.io — mainnet DAI + mainnet sunDAI
sundai.local — testnet LEAP + testnet sunDAI
burner.leapdao.org and localhost — testnet LEAP
otherwise — mainnet DAI

File: src/App.js [hunk 1]
```diff
@@ -405,12 +440,12 @@ class App extends Component {
     setTimeout(this.longPoll.bind(this),150)
 
     // NOTE: Change this to mainnet again when ready for mainnet launch.
-    let mainnetweb3 = new Web3(new Web3.providers.WebsocketProvider("wss://rinkeby.infura.io/ws/v3/f039330d8fb747e48a7ce98f51400d65"));
+    let mainnetweb3 = new Web3(WEB3_PROVIDER);
```
Comment (TimDaub) [inline src/App.js]: Shouldn't this be `new Web3.providers.WebsocketProvider(WEB3_PROVIDER)`?

Comment (troggy) [reply src/App.js]: to be of my knowledge, Web3 determines the provider to use automatically

Comment (TimDaub) [reply src/App.js]: Cool, TIL :)

**Comment by TimDaub** [comment]
I'll close this as it doesn't belong in master anymore.

**Pull request closed without merging**